admin-portal/src/app/_components/OrganizationSignUpForm.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+
+import { useState } from 'react';
+import axios from "axios";
+
+
+export default function NewOrganizationForm() {
+    const [community, setCommunity] = useState('');
+    const [description, setDescription] = useState('');
+    const [location, setLocation] = useState('');
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+
+    const apiEndpoint = `http://${process.env.IP_ADDRESS}:${process.env.PORT}/org/createOrg`;
+
+    const handleSubmit = async () => {
+        if (password !== confirmPassword) {
+            return;
+        }
+        const formData = { community:community, description:description, location:{type: "Point", coordinates: [0, 0, 0]}, name:name, password:password, confirmPassword:confirmPassword, };
+        try {
+            const response = await axios.post(apiEndpoint, formData);
+            console.log('Response:', response.data);
+            setCommunity('');
+            setDescription('');
+            setLocation('');
+            setName('');
+            setPassword('');
+            setConfirmPassword('');
+        } catch (error) {
+            console.error('Error submitting form:', error);
+        }
```

Comment:
Prevent refresh from occurring after form submission using the event object as well as `preventDefault`!